packages/contracts-periphery/deploy/op-nft/AttestationStationProxy.ts

```diff
@@ -61,9 +61,9 @@ const deployFn: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
     addr
   )
 
-  const implementation = await Proxy.callStatic.implementation({
-    from: ethers.constants.AddressZero,
-  })
+  const implementation = await Proxy.connect(
+    ethers.constants.AddressZero
+  ).callStatic.implementation()
```

Comment:
The previous call 
`Proxy.callStatic.implementation({
    from: ethers.constants.AddressZero,
  })`
  
 was throwing an error
```
Error: Contract with a Signer cannot override from (operation="overrides.from", code=UNSUPPORTED_OPERATION, version=contracts/5.7.0)
```